ui/components/KubeStatusIndicator.tsx

```diff
@@ -146,15 +146,15 @@ function KubeStatusIndicator({
     if (ready === ReadyType.Reconciling) {
       readyText = ReadyType.Reconciling;
       icon = IconType.ReconcileIcon;
-      iconColor = "primary";
+      iconColor = "primaryOriginal";
```

Comment:
This one should stay as `primary` I think.

We could type it as we do in https://github.com/weaveworks/weave-gitops/blob/de7ed81057ee4dc497038062c4aa186d3a53b2ab/ui/components/PageStatus.tsx#L21